packages/apple-pay-integration/src/apple-pay-button-strategy.ts

```diff
@@ -227,9 +286,44 @@ export default class ApplePayButtonStrategy implements CheckoutButtonStrategy {
             this._onPaymentAuthorized(event, applePaySession, paymentMethod);
     }
 
+    private async _createBuyNowCart() {
+        try {
+            const cartRequestBody = this._buyNowInitializeOptions?.getBuyNowCartRequestBody?.();
+
+            if (!cartRequestBody) {
+                throw new MissingDataError(MissingDataErrorType.MissingCart);
+            }
+            const buyNowCart = await this._paymentIntegrationService.createBuyNowCart(
+                cartRequestBody,
+            );
+            await this._paymentIntegrationService.loadCheckout(buyNowCart.id);
+        } catch (error) {
+            throw new BuyNowCartCreationError();
+        }
+    }
+
+    private _handlePaymentMethodSelected(applePaySession: ApplePaySession) {
+        const state = this._paymentIntegrationService.getState();
+        const cart = state.getCartOrThrow();
+        const config = state.getStoreConfigOrThrow();
+        const checkout = state.getCheckoutOrThrow();
+
+        if (!this._paymentMethod || !this._paymentMethod.initializationData) {
+            throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+        }
+
+        const request = this._getBaseRequest(cart, checkout, config, this._paymentMethod);
+        delete request.total.type;
```

Comment:
Should this line be here?

Reply:
yes, it has type pending and price doesn't displayed